package.json

```
@@ -1681,5 +1681,6 @@
     "xmlbuilder": "13.0.2",
     "yargs": "^15.4.1",
     "yarn-deduplicate": "^6.0.2"
-  }
+  },
+  "packageManager": "yarn@1.22.19"
```

To match `engines.yarn`: 
```suggestion
  "packageManager": "^yarn@1.22.19"
```

---

Ah didn't know you could do that. Sounds good!